analysis_options.yaml

```diff
@@ -420,7 +421,7 @@ linter:
     # recommendation: optional
     # reason: Can get annoying for React component lifecycle methods, constructors.
     # 0 issues
-    - public_member_api_docs
+    #- public_member_api_docs
```

Comment:
I find this one pretty annoying because it requires comments on constructors so it creates a ton of analyzer noise.